src/components/DeliverablesTable/index.tsx

```diff
@@ -41,34 +44,65 @@ const DeliverablesTable = ({
 }: DeliverablesTableProps) => {
   const dispatch = useAppDispatch();
   const { activeLocale } = useLocalization();
+  const { availableProjects: projects } = useProjects();
+  const { isAcceleratorRoute } = useAcceleratorConsole();
 
   const [deliverables, setDeliverables] = useState<ListDeliverablesElement[]>([]);
   const [deliverablesSearchRequestId, setDeliverablesSearchRequestId] = useState('');
   const deliverablesSearchRequest = useAppSelector(selectDeliverablesSearchRequest(deliverablesSearchRequestId));
 
-  const featuredFilters: FilterConfig[] = useMemo(
-    () =>
-      activeLocale
-        ? [
-            {
-              field: 'status',
-              // These options are strings for now, but may end up as enums when the BE types come through, if that is
-              // the case we will need to implement the renderOption and pillValueRenderer to render the desired
-              // human readable values
-              options: DeliverableStatuses,
-              label: strings.STATUS,
-            },
-            {
-              field: 'category',
-              // Same note as above
-              options: DeliverableCategories,
-              label: strings.CATEGORY,
-            },
-          ]
-        : [],
-    [activeLocale]
+  const getProjectName = useCallback(
+    (projectId: number) => (projects?.find((project: Project) => project.id === projectId) || {}).name || '',
+    [projects]
   );
 
+  const featuredFilters: FilterConfig[] = useMemo(() => {
+    const filters: FilterConfig[] = [
+      {
+        field: 'status',
+        // These options are strings for now, but may end up as enums when the BE types come through, if that is
+        // the case we will need to implement the renderOption and pillValueRenderer to render the desired
+        // human readable values
+        options: DeliverableStatuses,
+        label: strings.STATUS,
+      },
+      {
+        field: 'category',
+        // Same note as above
+        options: DeliverableCategories,
+        label: strings.CATEGORY,
+      },
+    ];
+
+    // show the project filter only in the accelerator route
+    // the participant view already has a projects filter above the table
+    if (isAcceleratorRoute) {
+      filters.unshift({
+        field: 'project_id',
+        options: (projects || [])?.map((project: Project) => `${project.id}`),
+        searchNodeCreator: (values: (number | string | null)[]) => ({
+          field: 'projectId',
+          operation: 'field',
+          type: 'Exact',
+          values: values.map((value: number | string | null): string | null => (value === null ? value : `${value}`)),
+        }),
+        label: strings.PROJECTS,
+        renderOption: (id: string | number) => getProjectName(Number(id)),
+        notPresentFilterShown: true,
```

Comment:
You can get rid of this and the next line, "no project" shouldn't be an option here.